test/integration/util_test.go

```diff
@@ -59,3 +61,41 @@ func UniqueProfileName(prefix string) string {
 	// example: prefix-20200413162239-3215
 	return fmt.Sprintf("%s-%s-%d", prefix, time.Now().Format("20060102150405"), os.Getpid())
 }
+
+// auditLastLine returns the last line of the audit.go log.
+// To make the runtime constant, even with a large file, it starts at the end and goes backwards
+func auditLastLine() (string, error) {
```

Comment:
not sure if this is needed (but I could be convinced otherwise)

how about just a simple checking the audit log for existence of a "command"

Reply:
I am currently only using this function in serial tests, so the parallelization is not an issue, but if this was used in a parallel test there would definitely be a race condition.

The issue I see with checking for the existence of a command is that `audit.json` is an ongoing log file, so assuming at least a single command is run there will always be the existence of a command. For example, the log file already has several commands in it already, someone modifies something so new commands are not logged, the test will still pass because there is a command present in the log file.

With that said, if I'm assuming correctly that machines that run this in CI start fresh without the file, then the failure should be detected then.

Let me know what you think @medyagh

Reply:
refer reponse to the gvc comments, better to be "parallel safe" and using the distnitive information to get logs
for example "profile" name.

I always try to push back on any integration test that starts to get complex, the integration test code should be dumb simple

Reply:
Good point, pivoted to this method instead